src/ex2/item_manager.js

```diff
@@ -0,0 +1,53 @@
+import PokemonClient from "./PokemonClient.js";
```

Comment:
Use the same conventions for file names, you file here is called "item_manager.js" while you also have "PokemonClient.js"